pkg/cmd/repo/create/create.go

```diff
@@ -564,7 +564,7 @@ func createFromLocal(opts *CreateOptions) error {
 
 func sourceInit(gitClient *git.Client, io *iostreams.IOStreams, remoteURL, baseRemote string) error {
 	cs := io.ColorScheme()
-	remoteAdd, err := gitClient.Command(context.Background(), "remote", "add", baseRemote, remoteURL)
+	remoteAdd, err := gitClient.AuthenticatedCommand(context.Background(), "remote", "add", baseRemote, remoteURL)
```

Comment:
`git remote add` without `-f` does not use the network and therefore should not need AuthenticatedCommand